NuGet.Config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="AspNetVNext" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
```

Comment:
Actually, it looks like this project doesn't need anything from the nightly feed. I'll just remove altogether.